packages/core/src/modules/proofs/ProofsApiOptions.ts

```diff
@@ -94,6 +94,7 @@ export interface RequestProofOptions<PPs extends ProofProtocol[] = ProofProtocol
  */
 export interface AcceptProofRequestOptions<PPs extends ProofProtocol[] = ProofProtocol[]> extends BaseOptions {
   proofRecordId: string
+  hasReturnRoute?: boolean
```

Comment:
hasReturnRoute gives me the idea of something that is already decided (so the name of a boolean), but the user can make the choise when calling this method.

Maybe we can call it `enableReturnRoute` or `useReturnRoute`?